docs/topics/maven.md

```diff
@@ -23,6 +23,26 @@ To use JDK 17, in your `.mvn/jvm.config` file, add:
 --add-opens=java.base/java.io=ALL-UNNAMED
 ```
 
+## Declare repositories
+
+To access artifacts in repositories that can be consumed by Maven builds, specify the ID and URL of each repository in
```

Comment:
Actually, according to the [Maven documentation](https://maven.apache.org/pom.html#Repositories), Maven Central is the default repository for any project. I think we should reflect this in this section.